knimin/tests/test_ag_pulldown.py

```diff
@@ -53,6 +55,13 @@ def test_get(self):
             self.assertIn("<option value='%s'>%s</option>" % (survey, survey),
                           response.body)
         self.assertNotIn('<input type="submit" disabled>', response.body)
+        for (id, name, selected) in db.list_ag_surveys():
```

Comment:
I think in this case none are selected, correct? according to the `get` call that you are doing all should be false.